core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/backoff/BackoffStrategy.java

```diff
@@ -21,8 +21,12 @@
 import software.amazon.awssdk.core.retry.RetryMode;
 import software.amazon.awssdk.core.retry.RetryPolicyContext;
 
+/**
+ * @deprecated Use instead {@link software.amazon.awssdk.retries.api.BackoffStrategy}
+ */
 @SdkPublicApi
```

Comment:
SdkPublicApi can be removed if its deprecated. Generic comment for other classes

Reply:
See above, the build **requires** each class to have one of the API scope annotations. If we remove this one, which one should we use?